specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2021-06-01-preview/Rules.json

```diff
@@ -336,9 +336,6 @@
         "compatibilityLevel": {
           "format": "int32",
           "type": "integer",
-          "default": 20,
-          "minimum": 20,
-          "maximum": 20,
           "description": "This property is reserved for future use. An integer value showing the compatibility level, currently hard-coded to 20."
         },
```

Comment:
@v-Ajnava Also for eventhubs since this is removed, would there be new examples reflecting the new change or thats optional?

Reply:
its optional and we don't have min/max in examples.

Reply:
as recommended, removed  min/max and description will have the range.